config/server.lua

```diff
@@ -113,6 +113,20 @@ return {
             end
         },
     },
+    logging = {
+        enableOxLogging = false,
+        enableWebhooks = false,
+        webhook = {
+            ['default'] = '', -- default
+            ['joinleave'] = '', -- default
+            ['ooc'] = '', -- default
+            ['anticheat'] = '', -- default
+            ['playermoney'] = '', -- default
+        },
+        role = {
+            '@everyone' --Role to tag for logging data. Roles use <@%roleid> and users/channels are <@userid/channelid>
+        }
```

Comment:
Maybe add a comment that this is for core only and can't be used as central configuration for other resources? Feel like it could be confusing.

Also, do we really want to tag everyone by default for core logs?

Reply:
The `@everyone` tag was a carryover from smallresources. I'll leave it as an empty table for people to use if they want to.